portality/bll/services/application.py

```diff
@@ -365,6 +365,12 @@ def application_2_journal(self, application, manual_update=True):
             journal.set_owner(application.owner)
         journal.set_seal(application.has_seal())
 
+        b = application.bibjson()
+        if b.get_one_identifier("pissn") == "":
+            b.add_identifier("pissn", None)
+        if b.get_one_identifier("eissn") == "":
+            b.add_identifier("eissn", None)
```

Comment:
If we're adding new code here, we should be using the current methods, not the deprecated ones.  So this would be `b.pissn` rather than `b.get_one_identifier("pissn")`, and `b.eissn` instead of `b.get_one_identifier("eissn")`

Reply:
sure!